broker/src/main/java/io/zeebe/broker/system/monitoring/ProcessLifetimeMetrics.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.system.monitoring;
+
+import io.prometheus.client.Gauge;
+
+public class ProcessLifetimeMetrics {
```

Comment:
A bit unsure with this name. Maybe we can change it to BrokerStepMetrics or BrokerMetrics? What do you think?

Reply:
Hm, I like `BrokerStepMetrics`. It is not too broad as `BrokerMetrics` :)